eZ/Publish/SPI/Persistence/Content/Location/Trash/TrashedItemDeleteResultList.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * File containing the TrashedLocation class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Persistence\Content\Location\Trash;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+/**
+ * Struct containing accessible properties on TrashedLocation entities.
+ */
+class TrashedItemDeleteResultList extends ValueObject implements \IteratorAggregate
```

Comment:
SPI can if needed use elements from API, so if these structs / values are identical we could consider skipping having duplicates here.